src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -1308,6 +1299,9 @@ async def scale_services(
                         callback_url=endpoint.callback_url,
                         enqueue_only=True,
                         endpoint_id=endpoint.id,
+                        enable_sudo_session=user_id_row_mapping[
+                            endpoint.created_user
```

Comment:
You need to judge whether to allow the sudo at session by the privileges of `session_owner`, not the one who created the service. Usually those two will be identical but there is a rare chance that admin or superadmin will create the session operating the service on behalf of other user.
```suggestion
                            endpoint.session_owner
```